lib/src/main/kotlin/com/lemonappdev/konsist/api/ext/list/KoRepresentsTypeProviderListExt.kt

```diff
@@ -60,10 +57,7 @@ fun <T : KoRepresentsTypeProvider> List<T>.withoutRepresentedTypeOf(kClass: KCla
         kClass.qualifiedName?.let { type -> !it.representsType(type) } ?: true &&
```

Comment:
Note that this was not changed as the expected response for `null` is `true` and not `false` (almost fall for this one!)